lib/elements/multiselect.js

```diff
@@ -155,6 +156,9 @@ class MultiselectPrompt extends Prompt {
       this.handleSpaceToggle();
     } else if (c === 'a') {
       this.toggleAll();
+    } else if (Object.keys(this.hotkeys).includes(c)) {
+      this.hotkeys[c].handle();
+      this.abort();
```

Comment:
For my usecase, `abort`ing was the correct behavior. If you'd like to move forward with this, I'll find a way to give the dev more control over what happens after the handler.